File: sbtPlugin/src/sbt-test/sbt-swagger-play/play-scala/app/controllers/HomeController.scala
package controllers

import javax.inject._
import play.api._
import play.api.mvc._

import io.swagger.v3.oas.annotations._
import io.swagger.v3.oas.annotations.media._
import io.swagger.v3.oas.annotations.responses._

@Singleton
class HomeController @Inject()(val controllerComponents: ControllerComponents) extends BaseController {

  @Operation
  @ApiResponse(content = Array(new Content(schema = new Schema(implementation = classOf[String]))))
  def index() = Action { implicit request: Request[AnyContent] =>
    Ok("Index")
  }
}

File: sbtPlugin/src/sbt-test/sbt-swagger-play/play-java/app/controllers/HomeController.java
package controllers;

import play.mvc.*;

import io.swagger.v3.oas.annotations.*;
import io.swagger.v3.oas.annotations.media.*;
import io.swagger.v3.oas.annotations.responses.*;

public class HomeController extends Controller {

    @Operation(description = "Index Page")
    @ApiResponse(content = @Content(schema = @Schema(implementation = String.class)))
    public Result index() {
        return ok("Hello");
    }
}
